app/forms/feedback_form.rb

```diff
@@ -0,0 +1,72 @@
+class FeedbackForm
+  Section = Struct.new(:id, :format, :answers)
+  Answer = Struct.new(:key, :label, :id, :other) do
+    def formatted_id
+      return { id: id, other: true } if other
+
+      id
+    end
+  end
+
+  def name
+    :feedback
+  end
+
+  def id
+    25_473_840
+  end
+
+  def template
+    {
+      tasks: template_for(tasks),
+      ratings: template_for(ratings),
+      reasons: template_for(reasons),
+      comments: { id: 60_742_937, format: :text }
+    }
+  end
+
+  def tasks
+    Section.new(
+      60_742_936, :radio,
+      [
+        Answer.new('3', 'Yes', 505_487_572),
+        Answer.new('2', 'No', 505_487_573),
+        Answer.new('1', 'Partially', 505_487_574)
+      ]
+    )
+  end
+
+  def ratings
+    Section.new(
+      60_742_964, :radio,
+      [
+        Answer.new('5', 'Very satisfied', 505_488_046),
+        Answer.new('4', 'Satisfied', 505_488_047),
+        Answer.new('3', 'Neither satisfied nor dissatisified', 505_488_048),
+        Answer.new('2', 'Dissatisfied', 505_488_049),
+        Answer.new('1', 'Very dissatisfied', 505_488_050)
+      ]
+    )
+  end
+
+  def reasons
+    Section.new(
+      60_745_386, :checkboxes,
+      [
+        Answer.new('3', 'Submit an LGFS claim', 505_511_336),
```

Comment:
Grammar: `an` should be `a`

Reply:
Is that correct? I would read it as "an el-gee-ef-es claim".

Reply:
https://www.a-or-an.com/a_an/l
🤔

Reply:
"Submit an LGFS claim" would appear to be grammatically correct.